packages/camera/camera_platform_interface/lib/src/platform_interface/camera_platform.dart

```diff
@@ -88,8 +88,9 @@ abstract class CameraPlatform extends PlatformInterface {
 
   /// Starts a video recording.
   ///
+  /// The length of the recording can be limited by defining the [maxVideoDuration]
```

Comment:
nit: technically the parameter is already defined, in this case it should be "specifying". There is also a period missing and it would be good to document the default behaviour:

```suggestion
  /// The length of the recording can be limited by specifying the [maxVideoDuration]. By default no maximum duration is specified, meaning the recording will continue until manually stopped.
```